packages/framework/src/Model/Mail/Mailer.php

```diff
@@ -99,45 +74,37 @@ protected function getMessageWithReplacedVariables(MessageData $messageData)
             $messageData->subject,
             $messageData->variablesReplacementsForSubject
         );
-        $fromEmail = $messageData->fromEmail;
-        $fromName = $messageData->fromName;
 
-        $message = new Swift_Message();
-        $message->setSubject($subject);
-        $message->setFrom($fromEmail, $fromName);
-        $message->setTo($toEmail);
+        $email = new Email();
+        $email
+            ->subject($subject)
+            ->from(new Address($messageData->fromEmail, $messageData->fromName))
+            ->to($messageData->toEmail)
+            ->text(htmlspecialchars_decode(strip_tags($body), ENT_QUOTES))
+            ->html($body);
         if ($messageData->bccEmail !== null) {
-            $message->addBcc($messageData->bccEmail);
+            $email->addBcc($messageData->bccEmail);
         }
         if ($messageData->replyTo !== null) {
-            $message->addReplyTo($messageData->replyTo);
+            $email->addReplyTo($messageData->replyTo);
         }
-        $message->setContentType('text/plain; charset=UTF-8');
-        $message->setBody(htmlspecialchars_decode(strip_tags($body), ENT_QUOTES), 'text/plain');
-        $message->addPart($body, 'text/html');
 
         foreach ($messageData->attachments as $attachment) {
-            try {
-                $swiftAttachment = Swift_Attachment::fromPath(
-                    $this->mailTemplateFacade->getMailTemplateAttachmentFilepath($attachment)
-                );
-            } catch (FileNotFoundException $exception) {
-                $this->logger->error('Attachment could not be added because file was not found.', [$exception]);
-                continue;
-            }
-            $swiftAttachment->setFilename($attachment->getNameWithExtension());
-            $message->attach($swiftAttachment);
+            $email->attachFromPath(
+                $this->mailTemplateFacade->getMailTemplateAttachmentFilepath($attachment),
+                $attachment->getNameWithExtension()
```

Comment:
seems like the exception is raised after all... 
I deleted file from the `project-base/web/content/uploadedFiles/mailTemplate` and from `project-base/src/var/fileUploads` and this exception occurred after trying to send email 
![Snímek obrazovky 2022-06-21 v 13 10 37](https://user-images.githubusercontent.com/1177414/174786625-8c8be0e1-2aad-4a42-97dc-e814e6c3f290.png)

Reply:
ah, thanks! Fixed in af1657584